package.json

```diff
@@ -12,22 +12,22 @@
     "url": "https://bitbucket.org/ilikeducks/duckling.git"
   },
   "dependencies": {
-    "grunt": "~0.4.5",
-    "grunt-bower-task": "^0.4.0",
-    "grunt-contrib-coffee": "^0.13.0",
-    "grunt-contrib-concat": "~0.5.1",
-    "grunt-contrib-copy": "~0.8.0",
-    "grunt-contrib-jade": "~0.14.1",
-    "grunt-contrib-sass": "~0.9.2",
-    "grunt-contrib-watch": "~0.6.1",
-    "grunt-image": "~1.0.6",
-    "grunt-tsd": "^0.1.0",
-    "grunt-typescript": "~0.6.2",
-    "jade": "~1.11.0",
-    "jasmine-core": "^2.3.4",
-    "nw": "~0.12.2",
+    "grunt": "*",
```

Comment:
The versions should be locked.  If they are *, then it is easy for our builds to get out of sync.